apps/server/src/modules/tool/context-external-tool/controller/api-test/tool-context.api.spec.ts

```diff
@@ -407,4 +410,68 @@ describe('ToolContextController (API)', () => {
 			});
 		});
 	});
+
+	describe('[GET] tools/context-external-tools/:contextExternalToolId', () => {
+		const setup = async () => {
```

Comment:
i guess it doesn't matter if there is only one describe-when, but we could think about unifying the place of the setup. Inside describe-when or upmost in the hirarchy (is there something in the style guide about this?)